doc/whats-new.rst

```diff
@@ -25,6 +25,13 @@ Enhancements
   raster files are opened with :py:func:`~xarray.open_rasterio`.
   By `Greg Brener <https://github.com/gbrener>`_
 
+- A new resampling interface to match pandas' group-by-like API was added to
+  :py:meth:`~xarray.Dataset.resample` and :py:meth:`~xarray.DataArray.resample`.
```

Comment:
Can you add a code snippet to show the difference? e.g., something like:

    Old syntax:

    In [1]: ds.resample('24H', dim='time', how='max')`
    [..]

    New syntax:

    In [2]: ds.resample(time='24H').max()
    [...]

The code doesn't even need to run.

Also please note that the old syntax is still supported for now, but with a warning.